src/schema/v2/homeView/sectionTypes/Artworks.ts

```diff
@@ -7,6 +7,11 @@ import { NodeInterface } from "../../object_identification"
 import { HomeViewGenericSectionInterface } from "./GenericSectionInterface"
 import { HomeViewSectionTypeNames } from "./names"
 import { standardSectionFields } from "./GenericSectionInterface"
+import { HomeViewSection } from "../sections"
+
+export interface HomeViewArtworksSection extends HomeViewSection {
```

Comment:
I am interested in hearing your thoughts about this type - we didn't need to export this so far because we always needed only one additional field that is hanled by a resolver. In this case that wasn't enough because of the new hard typed `enabledImpressionsTracking`. Please let me know if there is a better way to achieve this

Reply:
I think this makes sense to me! We're now diverging a bit from the parent interface in terms of configuration options so makes sense that we need to extend it.